yb-voyager/src/srcdb/common.go

```diff
@@ -44,7 +44,7 @@ func processImportDirectives(fileName string) error {
 	// Create a new scanner and read the file line by line.
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		line := strings.Trim(scanner.Text(), " ")
+		line:=scanner.Text()
```

Comment:
Create a separate PR for this issue. Generally, do not mix unrelated changes into same PR.

Reply:
Okay, raised.